packages/react/src/Uploader.tsx

```diff
@@ -150,14 +173,18 @@ export const UploaderRoot: Component<UploaderRootProps> = createComponent(
         () => [
           {
             file,
+            files,
             dataCID,
             status,
             error,
             handleUploadSubmit,
             storedDAGShards,
             uploadProgress
           },
-          { setFile: setFileAndReset }
+          {
+            setFile: (file?: File) => { setFilesAndReset((file != null) && [file]) },
```

Comment:
```suggestion
            setFile: (file?: File) => { file && setFilesAndReset([file]) },
```

Reply:
ah yea this is what I had originally, actually, but the linter wasn't happy. now tsc isn't happy. will find something that pleases them both, lol